package.json

```diff
@@ -30,7 +30,7 @@
     "@dojo/has": "beta1",
     "@dojo/i18n": "beta1",
     "@dojo/shim": "beta1",
-    "maquette": "2.4.3"
+    "maquette": "2.5.0"
```

Comment:
@mwistrand but @agubler is suggesting making it a dependency, which raised the question in my mind of how that would be resolved with WebPack.  So the only case it would be _caveat emptor_ then would be under and AMD loader.  I guess considering that is the case, and we feel WebPack is going to be our primary end user environment, we can get rid of `peerDependencies` all together.  The main reason for it was to _do no harm_ when loading in a browser.  The thing we likely would need though is something that warned end users that duplicate packages were being included because semver dependencies could not be resolved, so people aren't scratching their head why their bundles are so big.